src/engine/events/event.ts

```diff
@@ -0,0 +1,3 @@
+export default class GameEvent {
+
+}
```

Comment:
Might of been thinking I needed to add stuff the base class. I will change this to an interface.